index.html

```diff
@@ -42,79 +44,231 @@ <h1>I’m Alexandra. Glad to see you!</h1>
-                fill="#505F79"
-              />
-            </svg></a></li>
-
-            <li><a href="#"><svg
-              width="20"
-              height="20"
-              viewBox="0 0 20 20"
-              fill="none"
-              xmlns="http://www.w3.org/2000/svg">
-              <path
-                d="M13.6444 3.03455C10.1556 3.03455 7.28857 5.86213 7.28857 9.37937V19.3104C7.28857 19.6897 7.59946 20.0001 7.97942 20.0001H11.5718C11.9518 20.0001 12.2627 19.6897 12.2627 19.3104V9.65524C12.2627 8.89661 12.8844 8.27592 13.6444 8.27592C14.4043 8.27592 15.0261 8.89661 15.0261 9.65524V19.3104C15.0261 19.6897 15.3369 20.0001 15.7169 20.0001H19.3093C19.6893 20.0001 20.0001 19.6897 20.0001 19.3104V9.37937C20.0001 5.89662 17.1677 3.03455 13.6444 3.03455Z"
-                fill="#505F79"
-              />
-              <path
-                d="M2.48705 0C1.10535 0 0 1.10345 0 2.48276C0 3.86207 1.10535 4.96552 2.48705 4.96552C3.86874 4.96552 4.97409 3.86207 4.97409 2.48276C4.97409 1.10345 3.86874 0 2.48705 0Z"
-                fill="#505F79"
+      
+        <ul class="social-media-icons">
+          <li>
+            <a href="#"><img src="images/icons/Twitter.png" alt="twitter-icon"></a>
+          </li>
+
+          <li>
+            <a href="#"><img src="images/icons/Linkedin.png" alt="linkedin-icon"></a>
+          </li>
+
+          <li>
+            <a href="#"><img src="images/icons/Medium.png" alt="medium-icon"></a>
+          </li>
+
+          <li>
+            <a href="#"><img src="images/icons/Git.png" alt="github-icon"></a>
+          </li>
+
+
+          <li>
+            <a href="#"><img src="images/icons/AngelList.png" alt="angelist-icon"></a>
+          </li>
+        </ul>
+      </div>
+    </section>
+
+    <!----------------- Start of Portfolio works  ----------->
+
+    <section class="works_container">
+      <div class="card_works">
+        <img
+          src="images/snapshot/Snapshoot Portfolio.png"
+          alt="Snapshoot Portfolio"
+          class="portfolio_img"
+        />
+        <h2 class="card-text_title">Tonic</h2>
+        <div class="cards frame2">
+          <img src="images/frames/Frame 2.png" alt="frame2-img" />
+        </div>
+        <p class="description-text">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="languages_list">
+          <li class="html">HTML</li>
+          <li class="css">CSS</li>
+          <li class="javascript">Javascript</li>
```

Comment:
- [ ] Nice one following the design guidelines as best as possible, to make it more beautiful kindly add the little border around the cards to enable a 💯 pixel-perfect design 😃.

- Your design
<img width="151" alt="wrong" src="https://user-images.githubusercontent.com/41163571/227900133-c527b730-85a0-4393-b33c-ce88994ea685.png">

- Figma design
<img width="254" alt="right" src="https://user-images.githubusercontent.com/41163571/227900072-93ed1305-0174-4974-be1a-9424573f6124.png">